tests/test_markdown_extensions.py

```diff
@@ -61,19 +68,37 @@ def test_basic(self):
             '<h2 id="paragraph-header">Paragraph Header</h2>\n'
             '<p>Text</p>'
         )
+        codehilite = pages.get('codehilite') #Test codehilite also loaded
+        self.assertEqual(
+            codehilite.html,
+            '<div class="codehilite"><pre><span></span><span class="k">print</span>'
+            '<span class="p">(</span><span class="s1">&#39;Hello, world!&#39;'
+            '</span><span class="p">)</span>\n</pre></div>'
+        )
 
     def test_codehilite_linenums_disabled(self):
+        #Test implicitly disabled
```

Comment:
Not sure that I get the point here. Can you elaborate?

Reply:
The idea was to test using the default config, which should load codehilite with linenums disabled; not sure that it's necessary on second thought as we already have the fixture in test_basic?